server/routes/pages.js

```diff
@@ -32,13 +32,76 @@ module.exports = function(server)
                 footer: 'partials/footer',
             },
             stars: helper.generateStars(40),
-            clouds: helper.generateClouds(),
+            clouds: helper.generateClouds(1),
             meta: "Wetfish is a Free Culture community founded in 2005 that advocates personal privacy and organizational transparency."
         };
         
         event.emit('render', req, res, options);
     });
 
+
+    //match requested url with the news data url
+    function findPost(url)
+    {
+       //const formatted = url.replace('/post/','');
+        /*config.news.forEach(function(i,index, array)
+        {
+           
+            if (formatted==i['url']) 
+            {
+                //console.log(i);
+                console.log('found it!!!');
+                console.log(array);
+                return array;
+            }
+             
+        });
+       */
+       for (var i = 0; i < config.news.length; i++) 
+        {
```

Comment:
The indenting / line breaks here is still not consistent, please use the same style as the rest of the project.

Reply:
I thought i did, perhaps theirs a build tool for that i could use? but i think i can see the for is slightly off. i can move that no problem.